heron/api/src/java/org/apache/heron/streamlet/Streamlet.java

```diff
@@ -22,6 +22,8 @@
 import java.util.List;
 
 import org.apache.heron.classification.InterfaceStability;
+import org.apache.heron.streamlet.impl.operators.IStreamletBasicOperator;
```

Comment:
Given all the existing `StreamletOperator` extends `BaseRichBolt`, should we also enforce custom operator to be `BaseRichBolt`?

Thus we can reduce this redundant `IStreamletBasicOperator`.

Reply:
Yesh. I think we should enforce all StreamletOperator to be IRichBolt. CustomOperator will be the base class for all user defined operators (non-windowed onces) hence they will be IRichBolt.

IStreamletBasicOperator is not for CustomOperator. It is for reusing existing user Bolts (the ones extend from IBasicBolt instead of IRichBolt) only. That is why we have only IStreamletBasicOperator but no CustomBasicOperator.